### PR TITLE
[_ASPendingState] Make sure accessibility strings are not nil before allocating attributed strings for them #trivial

### PR DESCRIPTION
**Pull request by nguyenhuy**
- Fix crashes caused by https://github.com/TextureGroup/Texture/pull/554

### CODE REVIEW

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/5491aa4ad38507262fce1361bb06b138/raw/7c379dddd4c326416eaf60e5b686ca3266d32446/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/a4485fc5858b08d581ee3bca7b86e347/raw/8250e458181be0f7fcd6c33d91b939dabe01a257/log.txt)

File: Tests/ASDisplayNodeTests.mm [hunk 1]
```diff
@@ -353,7 +353,7 @@ - (void)checkValuesMatchDefaults:(ASDisplayNode *)node isLayerBacked:(BOOL)isLay
   XCTAssertEqual((id)nil, node.accessibilityLabel, @"default accessibilityLabel is broken %@", hasLoadedView);
   XCTAssertEqual((id)nil, node.accessibilityHint, @"default accessibilityHint is broken %@", hasLoadedView);
   XCTAssertEqual((id)nil, node.accessibilityValue, @"default accessibilityValue is broken %@", hasLoadedView);
-  if (AS_AT_LEAST_IOS11) {
+  if (@available(iOS 11.0, *)) {
```
Comment (nguyenhuy) [inline Tests/ASDisplayNodeTests.mm]: Switching to `@available` to make CI happy.

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/a3df344b1e3a71e9c0d68cb084a5fb22/raw/b0356b0dcbbccd464578d6857006acfdd60ba228/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/dd04f2dc8bf85f799b970765d3f559d5/raw/6b4c699690abdf8fa0fbf3cdd5dc15c070613016/log.txt)

File: Source/Private/ASDisplayNode+UIViewBridge.mm [hunk 2]
```diff
@@ -972,7 +972,7 @@ - (void)setAccessibilityHint:(NSString *)accessibilityHint
   _bridge_prologue_write;
   _setAccessibilityToViewAndProperty(_accessibilityHint, accessibilityHint, accessibilityHint, accessibilityHint);
   if (AS_AT_LEAST_IOS11) {
-    NSAttributedString *accessibilityAttributedHint = [[NSAttributedString alloc] initWithString:accessibilityHint];
+    NSAttributedString *accessibilityAttributedHint = accessibilityHint ? [[NSAttributedString alloc] initWithString:accessibilityHint] : nil;
```
Comment (maicki) [inline Source/Private/ASDisplayNode+UIViewBridge.mm]: What about using `accessibilityHint ?: [[NSAttributedString alloc] initWithString:accessibilityHint];`

File: Tests/ASDisplayNodeTests.mm (hunk 1 above)
Comment (maicki) [reply Tests/ASDisplayNodeTests.mm]: We're assuming everyone is using Xcode 9 now I guess?

Comment (nguyenhuy) [reply Tests/ASDisplayNodeTests.mm]: No, this is only for CI and contributors. Users aren't forced into Xcode 9.

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/f2b12fe5458e047b45b1f3f75687127b/raw/b512b33a02e6a7391ad06cf6bcc9f58bd7ed8e5e/log.txt)

**Pull request merged**